policies/expires-after-tagging/policy.json

```diff
@@ -20,11 +20,11 @@
           "notMatch": "####-##-##"
         },
         "then": {
-          "effect": "audit"
+          "effect": "deny"
```

Comment:
This is different from the enforcementMode. That is set here https://github.com/hmcts/azure-policy/blob/f9b8f39b639aa80affc693e6396d2f74cbc78576/assignments/mgmt-groups/mg-sds-sandbox/assign.expiresaftertag.json#L8